civihr_employee_portal/civihr_leave_absences/civihr_leave_absences.module

```diff
@@ -33,19 +33,30 @@ function civihr_leave_absences_block_view($delta = '') {
 }
 
 /**
+ * Implements hook_init().
+ *
+ * Fetches the base URL of the angular app, to be stored in the Drupal.settings global var
+ */
+function civihr_leave_absences_init() {
+  if (!_isCiviCRM()) {
+    $baseURL = CRM_Extension_System::singleton()->getMapper()->keyToUrl('uk.co.compucorp.civicrm.hrleaveandabsences');
```

Comment:
Basically usind keyToUrl() is fine. You can also use
CRM_Core_Resources::singleton()->getUrl('uk.co.compucorp.civicrm.hrleaveandabsences')
which is a bit shorter but anyway it calls keyToUrl() too but also allows to define specified file and add 'cache code' to avoid caching the script:
https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/Resources.php#L498
It also adds '/' to the end of the url.
Anyway in this case I think keyToUrl() fits better as it does exactly what you want.